### DISCUSSION TITLE
第2节课笔记(1班)

### DISCUSSION THREAD

**Discussion by KMnO4-zx** [InternLM/Tutorial#80] [笔记]
请在下方提交第 2 节课笔记笔记链接，知乎/CSDN/飞书 等等都可。
笔记让知识记得更牢固💪

**Comment by raulming** [comment]
https://quilled-apparatus-d9a.notion.site/377bb3c3463f402783813869a98df4b2

**Comment by Xpg74138** [comment]
https://blog.csdn.net/qq_41776136/article/details/135398320?csdn_share_tail=%7B%22type%22%3A%22blog%22%2C%22rType%22%3A%22article%22%2C%22rId%22%3A%22135398320%22%2C%22source%22%3A%22qq_41776136%22%7D

**Comment by xujinzh** [comment]
https://xujinzh.github.io/2024/01/06/ai-internlm-useage/

**Comment by woxinyonghen** [comment]
https://blog.csdn.net/xingchengyonghen/article/details/135422575

**Comment by wudemoai** [comment]
https://blog.csdn.net/weixin_48478550/article/details/135430892?spm=1001.2014.3001.5501

**Comment by RongjianLiang** [comment]
https://zhuanlan.zhihu.com/p/676422759

**Comment by biscuit279** [comment]
https://blog.csdn.net/qq_46212981/article/details/135429008

**Comment by blurmemo** [comment]
https://github.com/blurmemo/LLM/blob/main/note/InternLM/chap1.md

**Comment by Gu-Xiao** [comment]
https://www.bilibili.com/read/cv29102216/?jump_opus=1

**Reply by Gu-Xiao** [reply]
作业+笔记  https://blog.csdn.net/xidierzuo/article/details/135434824

**Comment by 10086zgyd** [comment]
https://blog.csdn.net/weixin_55546799/article/details/135441106?spm=1001.2014.3001.5502

**Comment by UncleLLD** [comment]
https://blog.csdn.net/uncle_ll/article/details/135440530

**Comment by AXYZdong** [comment]
https://nuly9zxzf1.feishu.cn/docx/B4rFdjWMuoiQXwxqqVuc5waynlb#ZLsGd8mswo1vBuxtBX1c3GMyn9f

**Comment by ustbzgn** [comment]
https://blog.csdn.net/qq_46052444/article/details/135433263?spm=1001.2014.3001.5502

**Comment by zpc-666** [comment]
https://blog.csdn.net/zpc1219/article/details/135440100

**Comment by gpt-dance** [comment]
https://zhuanlan.zhihu.com/p/676513982

**Comment by Andy1314Chen** [comment]
https://www.bilibili.com/read/cv29181045/?jump_opus=1

**Comment by gongxiucheng** [comment]
https://ckfajwu6zv.feishu.cn/docx/GuhFdjcnCoMJaZxCePbcA3zdnLg?from=from_copylink

**Comment by random-zhou** [comment]
http://t.csdnimg.cn/vbJHI

**Comment by grey2818** [comment]
https://zhuanlan.zhihu.com/p/676742628

**Comment by formorebeat** [comment]
https://shimo.im/docs/b30xIqjglk6rdBFJ/

**Comment by woml** [comment]
https://woml.github.io/Lesson_2

**Comment by xiaozhuang223** [comment]
https://zhuanlan.zhihu.com/p/676945852

**Comment by strong-leaf** [comment]
大模型(书生.浦语大模型)finetune实战_lesson2笔记与作业 - warm green的文章 - 知乎
https://zhuanlan.zhihu.com/p/677003853
